resources/pgbouncer.yaml

```diff
@@ -66,7 +66,7 @@ spec:
       serviceAccountName: stolon
       initContainers:
         - name: fix-secrets
-          image: quay.io/gravitational/stolon-pgbouncer:latest
+          image: stolon-pgbouncer:latest
```

Comment:
Why use `latest` here? that's usually an anti-pattern

Reply:
It does not matter here which version we set; it will be overwritten later by `tele build`